packages/abap-deploy-config-sub-generator/src/app/types.ts

```diff
@@ -0,0 +1,25 @@
+import type { AbapDeployConfigAnswersInternal } from '@sap-ux/abap-deploy-config-inquirer';
+import type { AppWizard } from '@sap-devx/yeoman-ui-types';
+import type { ConnectedSystem } from '@sap-ux/deploy-config-generator-shared';
+import type { TelemetryData } from '@sap-ux/fiori-generator-shared';
+
+export interface AbapDeployConfigOptions extends AbapDeployConfigAnswersInternal {
+    vscode?: unknown;
+    appWizard?: AppWizard;
+    launchDeployConfigAsSubGenerator?: boolean;
+    launchStandaloneFromYui?: boolean;
+    destinationRoot?: string;
+    projectPath?: string;
+    projectName?: string;
+    connectedSystem?: ConnectedSystem;
+    base?: string;
```

Comment:
What is this? Could we please add some docs on the options as these are what consumers need to pass in.

Reply:
the `base` file e.g `ui5.yaml`. I didn't like the name either but I can't update this as there is existing documentation to add this via ui5 tooling and command line. 
I will add comments